oophp-iban.php

```diff
@@ -37,15 +37,15 @@ public function HumanFormat() {
   return iban_to_human_format($this->iban);
  }
 
- public function Country($iban='') {
+ public function Country() {
   return iban_get_country_part($this->iban);
```

Comment:
@globalcitizen it's pretty clear from the diff of this commit, without any testing required, that those method parameters are not used and are misleading to have in the method signature.